xcode/Subconscious/Shared/Models/MemoAddress.swift

```diff
@@ -9,83 +9,81 @@ import Foundation
 
 enum MemoAddress: Hashable, Codable {
     case local(Slug)
-    case `public`(Slug)
+    case `public`(Slashlink)
```

Comment:
Public MemoAddress now holds a Slashlink, allowing for support of `@petname/slug` addresses in future.